docs/bisulfite-seq_test_data.md

```diff
@@ -1,13 +1,13 @@
 ### E. coli paired-end test dataset for Bisulfite-seq applications
```

Comment:
Delete this file - move not copy to the other dir 👍

Reply:
OK done